package.json

```diff
@@ -16,32 +16,45 @@
     "backbone": "~1.1.2",
     "bootstrap": "3.3.6",
     "bootstrap-switch": "3.3.2",
+    "colors": "0.6.2",
     "d3": "3.4.8",
     "eonasdan-bootstrap-datetimepicker": "~4.15",
+    "event-source": "0.1.1",
     "grunt": "~0.4.5",
     "grunt-cli": "~0.1",
     "grunt-contrib-concat": "~0.5.1",
+    "grunt-contrib-copy": "~0.5.0",
     "grunt-contrib-jade": "0.11.0",
     "grunt-contrib-stylus": "~0.22.0",
+    "grunt-contrib-symlink": "~0.3.0",
     "grunt-contrib-uglify": "0.11.0",
     "grunt-contrib-watch": "~0.6.1",
-    "grunt-contrib-copy": "~0.5.0",
-    "grunt-contrib-symlink": "~0.3.0",
-    "grunt-shell": ">=0.2.1",
-    "grunt-gitinfo": "~0.1.0",
     "grunt-file-creator": "~0.1.0",
     "grunt-fontello": "~0.3.1",
+    "grunt-gitinfo": "~0.1.0",
     "grunt-npm-install": "~0.2.0",
-    "remarkable": "1.6.2",
+    "grunt-shell": ">=0.2.1",
     "jade": "1.11.0",
     "jquery": "2.1.1",
+    "jsoneditor": "4.1.2",
+    "remarkable": "1.6.2",
     "sprintf-js": "1.0.2",
+    "string-replace": "^0.2.0",
     "swagger-ui": "2.0.22",
+    "toposort": "~0.2",
     "underscore": "~1.5",
-    "colors": "0.6.2",
-    "event-source": "0.1.1",
-    "jsoneditor": "4.1.2",
-    "toposort": "~0.2"
+    "vtk.js": "github:kitware/vtk-js",
+    "babel-core": "^6.11.4",
+    "dicom-parser": "^1.7.2",
+    "grunt-webpack": "^1.0.14",
+    "jade-loader": "^0.8.0",
+    "kw-web-suite": "^2.0.1",
+    "load-grunt-tasks": "^3.5.0",
+    "path": "^0.12.7",
+    "string-replace-loader": "^1.0.3",
+    "stylus-loader": "^2.1.2",
+    "webpack": "^1.13.1",
+    "webpack-dev-server": "^1.14.1"
```

Comment:
Some of these will probably need to be provided as "grunt.dependencies" in the `plugin.json` (some explanation [here](http://girder.readthedocs.io/en/latest/plugin-development.html#installing-custom-dependencies-from-npm)).  I also wouldn't be surprised if the post install step of vtk.js causes problems here as well.  I can provide some advice if that is the case.